spec/helpers/dns.lua

```diff
@@ -37,7 +37,10 @@ end
 
 --- Expires a record now.
 -- @param record a DNS record previously created
-function _M.dnsExpire(record)
+function _M.dnsExpire(client, record)
```

Comment:
What do you think about taking the opportunity and change all the camel case for sneak case? (I mean, other than `kong/resty/dns`, let's leave the legacy code alone 😄)

Reply:
I initially planned to fix all camel-case styles for balancer, DNS client, and related cases. However, this would make the PR even larger than it already is, exceeding 3000+ lines.

So, I believe it's better to address these style fixes separately. We can start by cleaning up the test cases. Additionally, we have a plan to review the balancer system again to find areas for optimization. Later on, we can clearly separate the balancer and DNS client functionalities.

Reply:
It makes sense, no need to rush everything on a single step. This one is already huge on its own.